source/accessories/amds/sensor-cards/current/index.md

```diff
@@ -36,43 +36,80 @@ It has an excellent bandwidth of 200khz and a low impedance current output that
+
+### Voltage Reference (LDO)
+The voltage reference, _V_<sub>_REF_</sub> is needed for the ADC. As 5V is readily available, and the LDO will have a minimum drop out voltage,  _V_<sub>_REF_</sub> = 4.5V was chosen. The LDO selected was `REF5045` from Texas Instruments, which can take a 5V input and provide a 4.5V reference output. This has an accuracy of 0.1% and low noise of 3μVpp/V.
 
 ### Op Amp Stage
-The voltage across the burden resistor is a bipolar signal (voltage span includes both positive and negative voltages).
-A non-inverting level translation circuit is designed using Op Amps as shown here:
+
+A non-inverting level translation circuit is implemented using Op Amps as shown here:
 
 ![](images/current-sensor-opamp-stage.svg)
 
-This circuit is used to translate the voltage across the burden resistor, which is bipolar, to the ADC input range of 0-4.5V. The resistor values can be calculated analytically. However, the algebra gets quite complicated. Hence it was computed using TI analog engineer's calculator. 
+This circuit is used to translate the voltage across the burden resistor, which is bipolar (voltage span includes both positive and negative voltages), to the ADC input range of 0-4.5V.
 
-**Note:** As the op-amp output voltage approaches the supply rails, it tends to distort and behave nonlinearly so the output voltage is limited to actually be 0.2V to 4.5V
+The output voltage for this circuit can be solved as:
 
-### Voltage Reference (LDO)
-The voltage reference, _V_<sub>_REF_</sub> is needed for the ADC. As 5V is readily available, and the LDO will have a minimum drop out voltage,  _V_<sub>_REF_</sub> = 4.5V was chosen. The LDO selected was `REF5045` from Texas Instruments, which can take a 5V input and provide a 4.5V reference output. This has an accuracy of 0.1% and low noise of 3μVpp/V.
+$$
+V_{\rm out} = \frac{R_{\rm a} R_{\rm b}}{R_{\rm a} R_{\rm b} + R_{\rm a} R_{\rm c} + R_{\rm b} R_{\rm c}} V_{\rm REF} + \frac{R_{\rm b} R_{\rm c}}{R_{\rm a} R_{\rm b} + R_{\rm a} R_{\rm c} + R_{\rm b} R_{\rm c}} V_{\rm BURDEN}
+$$
+
+A more precise expression for $V_{\rm BURDEN}$ can be derived as:
+
+$$
+V_{\rm BURDEN} = \frac{R_{\rm a} R_{\rm BURDEN}}{R_{\rm a} + R_{\rm BURDEN}} \left(\frac{N_1}{N_2}\right) I_{\rm PRIMARY} + \frac{R_{\rm BURDEN} }{R_{\rm a} + R_{\rm BURDEN}} V_{\rm out}
+$$
+
+The resistor values can be calculated from solving these expressions analytically. However, the algebra gets quite complicated. Instead, these values were computed using the [TI analog engineer's calculator](https://www.ti.com/tool/ANALOG-ENGINEER-CALC).
+
+The final design is implemented so that $I_{\rm PRIMARY} = 70A$ results in $V_{\rm out} \approx 0V$ and $I_{\rm PRIMARY} = -70A$ results in $V_{\rm out} \approx 5V$.
```

Comment:
Could these be opposite sign?

```suggestion
The final design is implemented so that $I_{\rm PRIMARY} = -70A$ results in $V_{\rm out} \approx 0V$ and $I_{\rm PRIMARY} = 70A$ results in $V_{\rm out} \approx 5V$.
```